tests/unit/steps/test_steps_plan.py

```diff
@@ -34,6 +34,24 @@ def generate_expected_upgrade_plan_principal(app, target, model):
         parallel=False,
         function=None,
     )
+    if app.charm in ["rabbitmq-server", "ceph-mon", "keystone"]:
```

Comment:
maybe adding a global variable to this list is going to be more clear

Reply:
These are only tests and I want to refactor it as I mentioned before to have proper fixtures. But I agree and I will create constant for it.